### PR TITLE
Zombie follow ab#304

### PR DESCRIPTION
**Pull request by kmangub**
- Added field 'areasVisited' in Player class
- Added field 'zombiesFollowing' in player class

### CODE REVIEW

File: src/main/java/com/engine/GameEngine.java [hunk 1]
```diff
@@ -216,6 +216,8 @@ private String getLookResult(String objectToFind) {
 
     public StringBuilder showStatus(String location) {
         StringBuilder builder = new StringBuilder();
+        System.out.println(player.getAreasVisited());
```
Comment (jmvaldez) [inline src/main/java/com/engine/GameEngine.java:219]: Was this meant to be appended to builder? or were these for testing?

File: src/main/java/com/engine/GameEngine.java [hunk 2]
```diff
@@ -216,6 +216,8 @@ private String getLookResult(String objectToFind) {
...
+        System.out.println("Zombies currently following you: " + player.getZombiesFollowing());
```
Comment (jmvaldez) [inline src/main/java/com/engine/GameEngine.java:220]: Was this meant to be appended to builder? or were these for testing?

File: src/main/java/com/engine/GameEngine.java (hunk 1 above)
Comment (kmangub) [reply src/main/java/com/engine/GameEngine.java:219]: The getAreasVisited is for visiting

File: src/main/java/com/engine/GameEngine.java (hunk 2 above)
Comment (kmangub) [reply src/main/java/com/engine/GameEngine.java:220]: Line 220 is for testing as well. I was thinking of incorporating that line to the 'look' command later on

File: src/main/java/com/engine/GameEngine.java (hunk 1 above)
Comment (jmvaldez) [reply src/main/java/com/engine/GameEngine.java:219]: Okay wasn't sure if these two print statements were needed

File: src/main/java/com/engine/GameEngine.java (hunk 2 above)
Comment (jmvaldez) [reply src/main/java/com/engine/GameEngine.java:220]: sounds good. ill approve

**Review by jmvaldez — APPROVED** [review]
Looks good

**Pull request merged**